We don't want to rebuild the target if it is not out of date

Signed-off-by: Jonas Karlsson <jonas@gobolinux,org>
--- albumart-1.6.2/Makefile.old	2008-03-24 18:45:00.000000000 +0100
+++ albumart-1.6.2/Makefile	2008-03-24 18:36:54.000000000 +0100
@@ -5,6 +5,6 @@
 all: uifiles
 
 uifiles:
-	@cd lib/albumart; for f in *.ui; do $(MAKE) -s -W "$$f"; done; cd ../..
+	@$(MAKE) -C lib/albumart
 .PHONY: uifiles
 
